maestrowf/interfaces/script/slurmscriptadapter.py

```diff
@@ -257,30 +257,24 @@ def _check_jobs_squeue(self, joblist, status):
         # squeue options:
         # -u = username to search queues for.
         # -t = list of job states to search for. 'all' for all states.
+        # -f = custom format options to guard against user customizations
 
+        squeue_fmt = "%.18i %.8j %.8u %.2t"
```

Comment:
yeah, good idea.   also noticed a '--parsable2' flag that might make at least the sacct variant a lot more robust -> no more dancing around sometimes empty columns.  Might do that on a subsequent pr since that's a little bit different parsing/version dependency.   There's also the json/yaml return types that might useful on some of these commands even though they're super verbose.